wpilib/tests/test_drive.py

```diff
@@ -248,16 +259,21 @@ def check_curvature(wpimock, drive_diff, y, rotation, isQuickTurn):
         elif rightMotorSpeed < -1.0:
             leftMotorSpeed -= rightMotorSpeed + 1.0
             rightMotorSpeed = -1.0
-    
+
     maxMagnitude = max(abs(leftMotorSpeed), abs(rightMotorSpeed))
     if maxMagnitude > 1.0:
         leftMotorSpeed /= maxMagnitude
         rightMotorSpeed /= maxMagnitude
 
-    drive_diff.leftMotor.set.assert_called_once_with(leftMotorSpeed * drive_diff.maxOutput)
-    drive_diff.rightMotor.set.assert_called_once_with(-rightMotorSpeed * drive_diff.maxOutput)
+    drive_diff.leftMotor.set.assert_called_once_with(
+        leftMotorSpeed * drive_diff.maxOutput
+    )
+    drive_diff.rightMotor.set.assert_called_once_with(
+        -rightMotorSpeed * drive_diff.maxOutput
+    )
 
 
+# fmt: off
```

Comment:
why is this one needed?

Reply:
I know. It looks like the code you're ignoring is the test cases on 280 - 289. any particular reason they are 4 to a line?

Reply:
Not really, but the formatting black applies is much worse IIRC.